jeeves/remind.py

```diff
@@ -28,14 +28,15 @@ def run_remind(config, blockers, server, header):
 	fpn = config.get('filter_param_name', None)
 	fpv = config.get('filter_param_value', None)
 	cac = config.get('cause_action_class', None)
+	filter_version = config.get('filter_version', None)
```

Comment:
Move default regex value(1{1}[0,3,6]{1}\.{1}\d{1}|1{1}[0,3,6]{1}(?=\D+)) here, instead of None. Otherwise  if filter_version is not in config get_osp_version will take None as filter_version argument.

Reply:
You're right . Thanks a lot